nexus/src/app/support_bundles.rs

```diff
@@ -90,7 +90,21 @@ impl super::Nexus {
             .db_datastore
             .zpool_get_sled_if_in_service(&opctx, bundle.zpool_id.into())
             .await?;
-        let client = self.sled_client(&sled_id).await?;
+
+        let short_timeout = std::time::Duration::from_secs(60);
+        let long_timeout = std::time::Duration::from_secs(3600);
```

Comment:
Should these be in the config file or something? It seems like it could, potentially, be useful to change them in the field if we see the long timeout hit...

Reply:
I'm not *opposed* to this, but I only wonder: is plumbing them into nexus config enough to make this modifiable? Seems like callers would still need to change the config among all nexuses, which would either require storing this value in the DB or deploying a new configuration via reconfigurator, right?